src/languages/es.ts

```diff
@@ -2641,7 +2641,7 @@ export default {
         someTagLevelsRequired: 'Falta etiqueta',
         tagOutOfPolicy: ({tagName}: ViolationsTagOutOfPolicyParams) => `La etiqueta ${tagName} ya no es válida`,
```

Comment:
This is not fixed yet

Reply:
I'll fix this on the back end so we can rely on `tagName` being sent to the client, without having to check if `tagName === 'Tag'